### PR TITLE
WIP feat: display dog pictures 

### PR DESCRIPTION
**Pull request by CatalinB7**

### CODE REVIEW

File: dogs_labeling/index.html [hunk 1]
```diff
@@ -10,21 +10,20 @@
...
-            <input type="text">
+        <form class="my-form">
+            <label for="name">Enter your name:</label>
+            <input type="text" id="name"> 
+            <button type="button" class="form-btn">Send it</button>
         </form>
     </div>
-
+    <div class="pics-container">
+        <div style="width: 100%; display: flex; justify-content: center; margin: 50px;">
```
Comment (anbalase) [inline dogs_labeling/index.html:20]: inline styles should be avoided due to their high specificity

File: dogs_labeling/index.html [hunk 2]
```diff
@@ -10,21 +10,20 @@
...
+            <button class="form-btn request-btn">My Silly Dogs</button>
```
Comment (anbalase) [inline dogs_labeling/index.html:21]: do you need both classes?

File: dogs_labeling/styles.css [hunk 3]
```diff
@@ -9,12 +9,93 @@ body {
...
+}
+
+.request-btn {
+    width: 15%;
+    font-size: 1.2rem;
+    font-weight: 500;
+    margin: 40px 50px 0px 50px;
+}
+
+/* for font-size should i use something that scales accordingly to the vw? */
```
Comment (anbalase) [inline dogs_labeling/styles.css:101]: why? 😄

File: dogs_labeling/script.js [hunk 4]
```diff
@@ -18,39 +36,38 @@ let changeAppearance = (data) => {
...
     mainContainer.style.display = "none";
 
-
-
-
-    let imgs = document.querySelectorAll("img");
-    for (let i = 0; i < nbPics; i++) {
-        imgs[i].onload = onLoadImg;
-        imgs[i].src = data.message[i];
+    //creating cards containing pics and 2 buttons
```
Comment (anbalase) [inline dogs_labeling/script.js:39]: instead of writing a comment maybe extract this to another function?

File: dogs_labeling/script.js [hunk 5]
```diff
@@ -1,12 +1,30 @@
...
+const noPics = 8;
 const picWidth = 300;
 let name = null;
-let clickedButton = async () => {
-    let data = await getData(nbPics);
+
+document.querySelector('.form-btn').addEventListener("click", clickedStartButton)
+document.querySelector('.refresh-btn').addEventListener("click", clickedRefreshButton)
+document.querySelectorAll('.request-btn')[0].addEventListener("click", clickedRequestButton);
+document.querySelectorAll('.request-btn')[1].addEventListener("click", clickedRequestButton);
```
Comment (anbalase) [inline dogs_labeling/script.js:8]: try out `Array.from(query).forEach(` 😄

File: dogs_labeling/index.html [hunk 6]
```diff
@@ -10,21 +10,20 @@
...
-
-
-
-<!--
-    js file loads at the end but i need the pics from beginning
-
-
- -->
\ No newline at end of file
+<script src="./script.js"></script>
```
Comment (alexdudescu) [inline dogs_labeling/index.html:28]: This should go inside the body tag

**Pull request closed without merging**